touchforms/formplayer/static/formplayer/script/fullform-ui.templates.js

```diff
@@ -0,0 +1,48 @@
+this["JST"] = this["JST"] || {};
+
+this["JST"]["formplayer/templates/formplayer/fullform-ui/form.html"] = function(obj) {
+obj || (obj = {});
+var __t, __p = '', __e = _.escape;
+with (obj) {
+__p += '<div>\n    <h1 id="title"></h1>\n    <div id="form"></div>\n    <input id="submit" type="submit" value="Submit" />\n</div>\n';
```

Comment:
Putting it in the fabfile seems like a nice idea. However, as we already discussed offline, it may be non-trivial considering this depends on npm, and getting npm to run as part of the deploy has its own set of issues.